src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1583,12 +1583,26 @@ public ModelEventCommand(string modelGuid, string eventName, int value = 1)
                 Value = value;
             }
 
+            [JsonConstructor]
+            public ModelEventCommand(string modelGuid, string eventName)
```

Comment:
```
public ModelEventCommand(Guid modelGuid, string eventName, int value = 1)
public ModelEventCommand(Guid modelGuid, string eventName)
```

@mjkkirschner How is this not ambiguous when attempting to call the second of these constructors?